### PR TITLE
treehouses services mariadb

### PR DESCRIPTION
**Pull request by vmnet8**
```
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services mariadb install
Pulling mariadb ... done
mariadb installed
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services mariadb up
mariadb_mariadb_1 is up-to-date
mariadb built and started
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services mariadb port
3306
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services mariadb size
275M
pi@treehouses:~/git_repo/cli $ sudo ./cli.sh services mariadb stop
Stopping mariadb_mariadb_1 ... done
mariadb stopped
```
......
please review.

### CODE REVIEW

**Comment by LordJashin32** [comment]
```
root@treehouses:/home/pi/cli# ./cli.sh services mariadb installPulling mariadb ... done
mariadb installed
root@treehouses:/home/pi/cli# ./cli.sh services mariadb up
Creating network "mariadb_default" with the default driver
Creating mariadb_mariadb_1 ... done
mariadb built and started
root@treehouses:/home/pi/cli# ./cli.sh services mariadb port
3306
root@treehouses:/home/pi/cli# ./cli.sh services mariadb size
275M
root@treehouses:/home/pi/cli# ./cli.sh services mariadb stop
Stopping mariadb_mariadb_1 ... done
mariadb stopped
root@treehouses:/home/pi/cli# ./cli.sh services mariadb down
Removing mariadb_mariadb_1 ... done
Removing network mariadb_default
^[[Amariadb stopped and removed
root@treehouses:/home/pi/cli# ./cli.sh services mariadb autorun true
service autorun set to true
root@treehouses:/home/pi/cli# ./cli.sh services mariadb autorun false
service autorun set to false
root@treehouses:/home/pi/cli# ./cli.sh services mariadb autorun
false
root@treehouses:/home/pi/cli# ./cli.sh services mariadb cleanup
Removing network mariadb_default
WARNING: Network mariadb_default not found.
Removing image jsurf/rpi-mariadb
mariadb stopped and removed
mariadb cleaned up

```
Seems to work for me. Cheers.

**Comment by dogi** [comment]
TODO:

- [ ] https://mariadb.org in description
- [ ] help.sh README.md
- [ ] connect to db test (best cli)
- [x] remove comment
- [ ] tab completion _treehouses

**Comment by dogi** [comment]
@kjong we need to think something about env variables like passwords

maybe  like in planet with volumes we add an password.yml to the docker compose ;)

File: services/install-mariadb.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  {
+    echo "version: '3'"
+    echo "services:"
+    echo "  mariadb:"
+    echo "    image: jsurf/rpi-mariadb"
+    echo "    container_name: rpi-mariadb"
+    echo "    ports:"
+    echo "      - 3306:3306"
+    echo "    environment:"
+    echo "      - MYSQL_ROOT_PASSWORD=my-secret-pw"
```
Comment (kjong) [inline services/install-mariadb.sh]: @dogi 

We can replace `    echo "      - MYSQL_ROOT_PASSWORD=my-secret-pw"` with `    echo "      - MYSQL_ROOT_PASSWORD=${password}"` and have a `.env` file with `password=`.

https://docs.docker.com/compose/env-file/

**Comment by dogi** [comment]
https://github.com/treehouses/cli/issues/871

**Pull request merged**